pkg/routing/interfaces.go

```diff
@@ -169,6 +169,7 @@ func ParticipantInitFromStartSession(ss *livekit.StartSession, region string) (*
 		AdaptiveStream:  ss.AdaptiveStream,
 		ID:              livekit.ParticipantID(ss.ParticipantId),
 		DisableICELite:  ss.DisableIceLite,
+		CreateRoom:      ss.CreateRoom,
```

Comment:
this is already contained in ParticipantInit, so we are rehydrating from here instead of passing in createRequest everywhere